ipmi-switches.yaml

```diff
@@ -0,0 +1,84 @@
+- name: "Gather required information"
+  hosts: ipmi_switches
+  gather_facts: false
+  tasks:
+
+    - name: "Calcuate rack and rackpos"
+      set_fact:
+        rack: "{{ parsed_address[-2] }}"
+        rackpos: "{{ parsed_address[-1] }}"
+      vars:
+        parsed_address: "{{ ansible_host | split('.') }}"
+
+    - name: "Create device role based on ansible group"
+      netbox.netbox.netbox_device_role:
+        netbox_url: "{{ netbox_url }}"
+        netbox_token: "{{ netbox_token }}"
+        data:
+          name: "{{ group_names[1] }}"
```

Comment:
hmm, I don't know what the right thing to do is. I would defer to you on this.
I could also just put "Generic" as device type and manufacturer since I don't think we have ever cared about the type of ipmi switches.

Reply:
this playbook is specifically for registering ipmi swithces, maybe I should just specify the role in the playbook instead of reading it from a variable?